### PR TITLE
Convert cheatsheet to JSON: DDG

### PR DESCRIPTION
**Pull request by kbhat95**
One step closer to resolving #1155 !

### CODE REVIEW

**Comment by moollaza** [comment]
@kbhat95 thanks a lot, if you wouldn't mind attaching a screenshot in the future, it would really help for these kinds of IA's that change the UI.

File: share/goodie/cheat_sheets/duckduckgo.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+    "id": "ddg_shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/duckduckgo.json]: please change this to `duckduckgo_cheat_sheet` to match the IA Page: https://duck.co/ia/view/duckduckgo_cheat_sheet

File: share/goodie/cheat_sheets/duckduckgo.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{
+    "id": "ddg_shortcuts", 
+    "name": "DuckDuckGo shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/duckduckgo.json]: Let's capitalize Shortcuts

**Comment by kbhat95** [comment]
@moollaza , sure. [Here](https://imgur.com/0kuIiH7) is a screenshot!

**Comment by moollaza** [comment]
@kbhat95 LGMT :+1: 

Thanks a lot for contributing, we really appreciate it :smiley:.

**Pull request merged**